Client/Components/TabManager.razor.cs

```diff
@@ -37,11 +37,46 @@ public partial class TabManager : IAsyncDisposable
         [CascadingParameter]
         public PageNotifications PageNotificationsComponent { get; set; }
```

Comment:
This could be made private as well (and all of the places we have CascadingParameters)

Reply:
Done